pkg/cvo/status.go

```diff
@@ -259,7 +260,11 @@ func (optr *Operator) syncStatus(original, config *configv1.ClusterVersion, stat
 			case len(validationErrs) > 0:
 				message = fmt.Sprintf("Reconciling %s: the cluster version is invalid", version)
 			case status.Fraction > 0:
-				message = fmt.Sprintf("Working towards %s: %.0f%% complete", version, status.Fraction*100)
+				tasks := make([]string, 0, len(status.Current))
+				for _, task := range status.Current {
+					tasks = append(tasks, task.KindName())
+				}
+				message = fmt.Sprintf("Working towards %s: %.0f%% complete (%s)", version, status.Fraction*100, strings.Join(tasks, ", "))
```

Comment:
The only thing that should ever show up in this message is 1-2 operator names, that's it.

Reply:
I don't want to keep showing operator names either.  The only time we should have anything other than a % is when a specific operator has taken longer than a certain amount of time (has completed a sync period)

Reply:
> The only thing that should ever show up in this message is 1-2 operator names, that's it.

So what to show when you're, say, blocked on a RoleBinding?  Collapse to the namespace?

Reply:
> The only time we should have anything other than a % is when a specific operator has taken longer than a certain amount of time (has completed a sync period)

The issue here is that the installer doesn't know about sync periods, and wants to know what was blocking the CVO when the installer decided to five up on waiting.  If the CVO starts right in on another sync cycle (which it does, right?), it seems very unlikely that the last message received by the installer would include blockers, unless we include them in every message.

Reply:
> I don't want to keep showing operator names either. The only time we should have anything other than a % is when a specific operator has taken longer than a certain amount of time (has completed a sync period)

This or the namespace is sufficient to find the next level of debugging.  We have enough operators that indicating which one the CVO is stuck on is pretty useful.  In cases where we aren't currently waiting on a CO, knowing the namespace is good enough for most purposes.